docs/data/sql_functions.yml

```diff
@@ -646,6 +646,9 @@ collection:
   - sql: ARRAY_UNION(array1, array2)
     table: haystack.arrayUnion(array)
     description: Returns an array of the elements in the union of array1 and array2, without duplicates. If any of the array is null, the function will return null.
+  - sql: ARRAY_CONCAT(array1, ...)
+    table: array1.arrayConcat(array1, ...)
+    description: Returns an array of the elements in the concat of at least one array. If all of the arrays are null, the function will return null.
```

Comment:
From the description it is not clear what happens if not every input array is `null`